kymatio/scattering1d/core/scattering1d.py

```diff
@@ -160,9 +151,7 @@ def scattering1d(x, pad, unpad, backend, log2_T, psi1, psi2, phi, pad_left=0,
                     else:
                         S_2 = unpad(U_2_m, ind_start[k1 + k2], ind_end[k1 + k2])
 
-                    out_S_2.append({'coef': S_2,
-                                    'j': (j1, j2),
-                                    'n': (n1, n2)})
```

Comment:
personally I found it nicer having this be on multiple lines than just one line, but thats just me

Reply:
should i undo?

Reply:
if you can that would be nice